descriptor.md

```diff
@@ -161,6 +162,14 @@ Implementations MAY implement SHA-512 digest verification for use in descriptors
 When the _algorithm identifier_ is `sha512`, the _encoded_ portion MUST match `/[a-f0-9]{128}/`.
 Note that `[A-F]` MUST NOT be used here.
 
+#### BLAKE3
+
+[BLAKE3][blake3] is a high performance, highly parallelizable, collision-resistant hash function which [is more performant][blake3-vs-sha2] than
+[SHA-256][rfc4634-s4.1]. The hash output length MUST be 256 bits. Implementations MAY implement BLAKE3 digest verification for use in descriptors.
```

Comment:
I think we normally split-on-sentence here (maybe @sudo-bmitch can confirm; he's been more pedantic/on top of that than I am):

```suggestion
[BLAKE3][blake3] is a high performance, highly parallelizable, collision-resistant hash function which [is more performant][blake3-vs-sha2] than [SHA-256][rfc4634-s4.1].
The hash output length MUST be 256 bits.
Implementations MAY implement BLAKE3 digest verification for use in descriptors.
```

Reply:
Agree with the split-on-sentence. It makes future PRs a lot easier to review (a one world change doesn't rewrap a paragraph).

Reply:
Fixed.